docs/user_manual/source/pipeline.rst

```diff
@@ -100,7 +110,16 @@ The |corev| experiences a 1 cycle penalty on the following hazards.
 
  * Load data hazard (in case the instruction immediately following a load uses the result of that load)
  * Jump register (jalr) data hazard (in case that a jalr depends on the result of an immediately preceding non-load instruction)
+ * Any implicit CSR read in ID (mret or table jump) while any implicit or explicit CSR access is in the WB stage
```

Comment:
Any implicit CSR read -> An instruction causing an implicit CSR read
any implicit or explicit CSR access -> a CSR access instruction or an instruction causing an implicit CSR access

Same type of conversion for following lines